### DISCUSSION TITLE
Drug with MW between 4 and 5kDa is considered as small or large molecule in PK-SIM?

### DISCUSSION THREAD

**Discussion by Gre-ux** [Open-Systems-Pharmacology/Forum#1579] [Questions & Problems]
Hello! I would like to know which model should be used when dealing with a peptide drug with molecule weight between 4kDa and 5kDa. Is it considered a small molecule or large molecule in PK-SIM?
I read the tutorials on the learn-esqlabs website. It says " Molecules are considered small up to a molecular weight of 5 kDa.". So I am a little confused.
Looking forward for your help!

**Comment by StephanSchaller** [comment]
Hi @Gre-ux, unfortunately, there is no general "true" value for the right threshold for when to switch to the large molecule model.

Starting from 0.5 kD to 5 kD it could already make sense. Just try it out and compare the outcomes.
Make sure to keep the property "is small molecule" checked/active so have passive diffusion into cells.

Best,
Stephan
